### DISCUSSION TITLE
server does not support  documentSymbolProvider

### DISCUSSION THREAD

**Discussion by Geobert** [nvim-java/nvim-java#220] [Q&A]
Hi, 

after installation, I have a bunch of messages that suggests everything is in order, but when I try to call Telescope’s current document’s symbols I get this message
```
[telescope.builtin.lsp_*]: server does not support  documentSymbolProvider  
```
and similar issue for go to definition but with `definitionProvider`

**Comment by s1n7ax** [comment]
@Geobert Indication of nvim-java being incorrectly configured somehow. Link to your config?

**Reply by Geobert** [reply]
Thanks for your answer, since my question, I change from kickstart to LazyVim, and the message I got is clearer: needs java version 17+ while I run on 11 :)

**Reply by s1n7ax** [reply]
Nvim-java downloads jdk for its use. You can configure jdtls to use Java 11 for development. Instructions are in the readme
